Modules/Filtering/ImageFrequency/test/itkFrequencyFFTLayoutImageRegionIteratorWithIndexTest.cxx

```diff
@@ -263,7 +263,7 @@ itkFrequencyFFTLayoutImageRegionIteratorWithIndexTest(int, char *[])
   constexpr unsigned int Dimension = 3;
 
   using CharPixelType = char;
-  using FloatPixelType = char;
+  using FloatPixelType = float;
```

Comment:
I'd dare to say that this change is unrelated to the `std::complex` warning. If that is the case, can it please be put into a separate commit so that the corresponding warning is clear, or even better, a separate PR so that it can be merged independently?

Reply:
I put it into a separate commit.